### DISCUSSION TITLE
specify rendering mode when using plot() method

### DISCUSSION THREAD

**Discussion by letaltom** [pyvista/pyvista#5148] [Q&A] [answered]
Is it possible to specify rendering mode (which can be switched using "Toggle rendering mode" button on the Jupyter widget) using an argument in plot() method (pyvista._plot) or perhaps a global settings before plot() call?

**Comment by tkoyama010** [comment, marked as answer]
@letaltom
Yes it is. Please check [Trame Jupyter Modes](https://tutorial.pyvista.org/tutorial/00_jupyter/index.html#trame-jupyter-modes) in the PyVista Tutorial.
```python
import pyvista as pv
pv.set_jupyter_backend('client')

pv.Cone().plot()
```
```python
import pyvista as pv
pv.set_jupyter_backend('trame')

pl = pv.Plotter()
pl.add_mesh(pv.Cone())
pl.show(jupyter_backend='client')
```

**Reply by letaltom** [reply]
Thank you. Although I have seen this argument in plot() method, the options listed in its help were just 'none', 'static' or 'trame'.
